driver-core/build.gradle

```diff
@@ -38,6 +39,14 @@ dependencies {
     testCompile project(':bson').sourceSets.test.output
 }
 
+buildConfig {
+    appName = 'mongo-java-driver'
+    version = getGitVersion()
+
+    clsName = 'MongoDriverVersion'
+    packageName = 'com.mongodb.internal.connection'
```

Comment:
`com.mongodb.internal.connection` seems the wrong package to me would: `com.mongodb.internal.build` be better?

Reply:
My intent was to put it in the same package as ClientMetadataHelper and not publicize its existence.  I was actually looking for a way to make it non-public but that would require a change to the plugin.